packages/core/src/app/billing/BillingForm.spec.tsx

```diff
@@ -15,6 +16,7 @@ import { DynamicFormField } from '../ui/form';
 import { getBillingAddress } from './billingAddresses.mock';
 import BillingForm, { BillingFormProps } from './BillingForm';
 import StaticBillingAddress from './StaticBillingAddress';
+import { getAddress } from '@bigcommerce/checkout/test-utils';
```

Comment:
I know right now our linter does not pick this up.
Could you please put this import in its right place?

Reply:
oh, thanks, missed this